app/src/main/java/org/sopt/sample/presentation/home/HomeViewModel.kt

```diff
@@ -0,0 +1,55 @@
+package org.sopt.sample.presentation.home
+
+import android.util.Log
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import org.sopt.sample.data.remote.ServicePool
+import org.sopt.sample.data.remote.response.ResponseUser
+import org.sopt.sample.util.state.UiState
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+
+class HomeViewModel : ViewModel() {
+
+    private val userService = ServicePool.userListService
+    private val _userResult: MutableLiveData<UiState<List<ResponseUser.userListInfo>>> =
+        MutableLiveData(UiState.Init)
+    val userResult: LiveData<UiState<List<ResponseUser.userListInfo>>>
+        get() = _userResult
+
+//    private val _userList = MutableLiveData<List<ResponseUser.userListInfo>>()
+//    val userList: LiveData<List<ResponseUser.userListInfo>>
+//        get() = _userList
+
+    init {
+        _userResult.value = UiState.Loading(true)
+    }
+
+    fun getUser() {
+        userService.getUserList().enqueue(object : Callback<ResponseUser> {
+            override fun onResponse(
+                call: Call<ResponseUser>,
+                response: Response<ResponseUser>
+            ) {
+                _userResult.value = UiState.Loading(false)
```

Comment:
```suggestion
    fun getUser() {
        _userResult.value = UiState.Loading()
        userService.getUserList().enqueue(object : Callback<ResponseUser> {
            override fun onResponse(
                call: Call<ResponseUser>,
                response: Response<ResponseUser>
            ) {
```
로딩을 위로 올려서, 요런 식으로 하면 좋을 것 같다고 생각이 듭니다..!